media-video/guvcview/guvcview-2.2.1.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Simple Qt6 or GTK+3 interface for capturing and viewing video from v4l2 devices"
+HOMEPAGE="https://guvcview.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/${PN}/source/${PN}-src-${PV}.tar.bz2"
+S="${WORKDIR}/${PN}-src-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl pulseaudio qt6 sfml"
```

Comment:
@a17r I'm blanking on if we prefer USE=qt for this case or not right now?

Reply:
@thesamesam guvcview-2.1.0 had IUSE=qt5. Dunno if upstream make any preference between GTK or Qt

Reply:
from git-sf : 
> guvcview will build with Gtk3 support by default